app/styles/app.styl

```diff
@@ -23,6 +23,9 @@ div[role='application']
     width: 100%
     display: flex
 
+*
```

Comment:
you can apply it on `path` only (rather than `*`), it's less agressive for CCSOM.